CONTRIBUTING.md

```diff
@@ -1,145 +1,125 @@
 
-The header is required and must not exceed 70 characters to ensure it is well-formatted in common git tools. It has a special format that includes a *type*, *scope* and *subject*:
+    ```bash
+    $ git clone https://github.com/<your-github-username>/one-app-router
+    $ cd one-app-router
+    ```
 
-Syntax:
+   > replace `your-github-username` with your github username
 
-```bash
-<type>(<scope>): <subject>
-```
+3. Install the dependencies by running
 
-#### Type
+    ```bash
+    $ cd npm install
+    ```
 
-The *type* should always be lowercase as shown below.
+4. You can now run any of these scripts from the root folder.
 
-##### Allowed `<type>` values:
+#### Running and cleaning the build files
 
-* **feat** (new feature for the user)
-* **fix** (bug fix for the user, not a fix to build scripts)
-* **docs** (changes to documentation)
-* **style** (formatting, missing semi colons, etc; no functional code change)
-* **refactor** (refactoring production code, eg. renaming a variable)
-* **test** (adding missing tests, refactoring tests; no production code change)
-* **chore** (updating build/env/packages, etc; no production code change)
+- `npm run build`
 
-#### Scope
+Runs `babel` to compile `src` files to transpiled JavaScript into `lib` using
+[`babel-preset-amex`](https://github.com/americanexpress/babel-preset-amex).
 
-The *scope* describes the affected code. The descriptor may be a route, component, feature, utility, etc. It should be one word or camelCased, if needed:
+#### Running tests
 
-```bash
-feat(transactions): added column for quantity
-feat(BalanceModule): initial setup
-```
+- `npm run lint`
 
-The commit headers above work well if the commit affects many parts of a larger feature. If changes are more specific, it may be too broad. To better clarify specific scopes, you should use a `feature/scope` syntax:
+Verifies that your code matches the American Express code style defined in
+[`eslint-config-amex`](https://github.com/americanexpress/eslint-config-amex).
 
-```bash
-fix(transaction/details): missing quantity field
-```
 
-The above syntax helps reduce verbosity in the _subject_. In comparison, consider the following example:
+- `npm test`
 
-```bash
-fix(transaction): missing quantity field in txn details
-```
+Runs unit tests **and** verifies the format of all commit messages on the current branch.
 
-Another scenario for scope is using a `route/scope` (or `context/scope`) syntax. This would be useful when a commit only affects a particular instance of code that is used in multiple places.
+- `npm posttest`
 
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
```

Comment:
this happens in `npm test`, there is no posttest for this library

Reply:
updated 42cb65f